app/models/vendor.rb

```diff
@@ -0,0 +1,17 @@
+class Vendor < ActiveRecord::Base
+  belongs_to :market
+  has_many :products
+  has_many :sales
+
+  def self.current_month
+    Time.now.month
+  end
```

Comment:
If its purpose is for display on the view, I'd put it in a helper file that handles date time kinds of tasks. I typically use helpers as a way to pull repetitive formatting logic out of the view. If you were going to use this method to help calculate things within the model, you could just make it a private method and leave it where it is.